extensions/extensions-helm/aissemble-spark-history-chart/templates/local-pv.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if eq .Values.eventVolume.storageType "local" }}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    type: local
+  name: {{ .Values.eventVolume.name }}-pv
+spec:
+  {{- with .Values.eventVolume.accessModes }}
+  accessModes:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  capacity:
+    storage: {{ .Values.eventVolume.size }}
+  hostPath:
+    path: {{ .Values.eventVolume.volumePathOnNode }}
+    type: DirectoryOrCreate
+  persistentVolumeReclaimPolicy: Delete
+  storageClassName: spark-event-storage
```

Comment:
Should this be conditional like it is in `events-pvc.yaml`?

Reply:
No.  The name here is hardcoded so that if local storage is selected, it can be immediately plugged in without any additional configuration necessarily being required.  You'll notice that the condition on this template as a whole matches the condition for setting the `events-pvc-yaml` `storageClassName` to the value we see here.